### PR TITLE
Add documentation for Data Enrichment

### LINKED ISSUE #91: Add documentation for Data Enrichment
Data Enrichment is a super useful feature when sending events to Keen IO. You can see more about it here: https://keen.io/docs/streams/data-enrichment-overview/. 

The JavaScript docs in the API Reference is a good example: https://keen.io/docs/api/?javascript#data-enrichment

They would be good to include in the section about sending events to Keen.

### PR DESCRIPTION
**Pull request by swapagarwal**
Close #91

### CODE REVIEW

**Comment by swapagarwal** [comment]
ping

**Comment by swapagarwal** [comment]
ping

File: README.md [hunk 1]
```diff
@@ -105,6 +105,67 @@ $event = ['purchase' => ['item' => 'Golden Elephant']];
...
+            [
+                'name' => 'keen:url_parser',
+                'input' => [
+                    'url' => 'page_url'
+                ],
+                'output' => 'parsed_page_url'
+            ]
+        ]
+    ]
+])
```
Comment (tbarn) [inline README.md]: Could you add semicolons at the end of each?

Comment (swapagarwal) [reply README.md]: My bad, added! 😄

**Comment by tbarn** [comment]
Thanks for the contribution @swapagarwal! 🎉  Could you fill out this form? https://goo.gl/forms/HgAERcDj3fkQL3vE3

We are low on some shirts, but we might be able to send it something else instead.

**Pull request merged**